src/CommonDBRelation.php

```diff
@@ -2145,4 +2146,45 @@ public static function getMemberPosition(string $class): int
             return 0;
         }
     }
+
+    public function check($ID, int $right, ?array &$input = null): void
+    {
+        $error_types = [];
+        $required_fields = [];
+        $must_check = $input !== null && ($right == CREATE || $right == UPDATE);
```

Comment:
IMHO, we should not have to pass all the relation specs during an update, as we may want to change an extra field value only. Changing the relation is probably not well supported, and I even wonder if we use a check for this somewhere in GLPI.

```suggestion
        $must_check = $input !== null && $right == CREATE;
```